### ISSUE TITLE
[BUG] Blender 2.91 not working with BlendNet

### ISSUE THREAD

**Issue by rabits** [state-of-the-art/BlendNet#125]
### Issue description:
Seems the blender devs are changed the interface and now blender binary is not located there: https://app.circleci.com/pipelines/github/state-of-the-art/BlendNet/249/workflows/e938693e-6a57-4e15-a71c-6d4dd9bd7a2b/jobs/881
```
DEBUG: Run subprocess "['/srv/blender/2.91/python/bin/python3.7m', '-b', '-noaudio', '-y', '/workspace/BlendNet_cache/ws/tmp7r6zt9x0testproject-2011271958-32-FEA_237/project/test-project.blend', '-P', '/srv/scripts/addons/blendnet/BlendNet/script-render.py', '--', 'config-render.json']" in "/workspace/BlendNet_cache/ws/tmp7r6zt9x0testproject-2011271958-32-FEA_237"
INFO: Starting process stderr read
>err>> Unknown option: -n
INFO: Starting process stdout read
>err>> usage: /srv/blender/2.91/python/bin/python3.7m [option] ... [-c cmd | -m mod | file | -] [arg] ...
INFO: Read of process stdout completed
>err>> Try `python -h' for more information.
Exception in thread Thread-3628:
Traceback (most recent call last):
  File "/srv/blender/2.91/python/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/srv/blender/2.91/python/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/srv/scripts/addons/blendnet/BlendNet/AgentTask.py", line 79, in _executionStderrWatcher
    for line in iter(process.stderr.readline, b''):
ValueError: readline of closed file
DEBUG: Return code: 2
```
```
/srv/blender/blender:15: RuntimeWarning: Use 'sys.executable' instead of 'binary_path_python'!
```

### Environment:
* **Application version**: 0.3.3
* **Blender client version**: 2.91.0
* **Blender worker version**: 2.91.0

**Comment by rabits** [comment]
About the `binary_path_python` - seems it become deprecated but was not fixed in blender internal:
```
intern/mantaflow/intern/strings/fluid_script.h:    sys.executable = bpy.app.binary_path_python\n";
```
So not related to the BlendNet anyhow. Seems fixed in blender master already.

**Comment by rabits** [comment]
Looks like BlendNet used wrong way to locate the blender path, so need to switch from sys.executable to `bpy.app.binary_path` as mentioned:
```
Blender's executable remains accessible via `bpy.app.binary_path`
```
And yeah, it's working for blender >= 2.80.

**Comment by rabits** [comment]
Ok, fixed in v0.3.4
